data/campaigns/Eastern_Invasion/ART_MUSIC_LICENSE.txt

```diff
@@ -0,0 +1,406 @@
+# AUDIO
+################################################################################################################################################################################
+//--------------------
+// MUSIC
+//--------------------
+cry from elensefar (pluft) - CC BY SA 4.0
+    https://forums.wesnoth.org/viewtopic.php?t=48565
+    https://soundcloud.com/pluft/cry-from-elensefar
+
+heavens (gabriel silver) - CC BY 4.0
+    remixed by dalas; I release my remix under CC BY 4.0
+    https://forums.wesnoth.org/viewtopic.php?t=48565
+    https://www.youtube.com/watch?v=AOvwWOGH7bQ&list=PLk-SdLbgd5ol62NX0q7ys5ZbmJFVAMzCF&index=2
+
+
+//--------------------
+// SOUNDS
+//--------------------
+wind (hackerb9) - CC 0
+    edited by dalas; I release the edited sound under CC BY 4.0
+    https://freesound.org/people/hackerb9/sounds/555246/
+
+gate-fall
+lich_die_wail
+magic-twilight
+magic-twilight-miss
+    made by dalas by modifying pre-existing wesnoth sounds
+    I release these under CC BY 4.0
+
+warhammer (Iwiploppenisse) - CC BY 4.0
+    https://freesound.org/people/Iwiploppenisse/sounds/156031/
+    modified by dalas by adding pre-existing wesnoth sounds
+    I release my modifications under CC BY 4.0
+
+
+
+
+
+################################################################################################################################################################################
+# ASSORTED IMAGES
+################################################################################################################################################################################
+//--------------------
+// ATTACKS
+//--------------------
+blast-wave
+sword-astral-short
+sword-orcish-flaming
+    made by dalas by modifying pre-existing wesnoth images
+    I release these under CC BY 4.0
+
+sword-orcish-swing
+    from the Reign of Lords add-on
+
+
+//--------------------
+// HALO
+//--------------------
+terror
+    created by dalas
+    I release these images under CC BY 4.0
+
+idle-flash
+fire-aura-small
+light-shield
+obscures-aura
+    made by dalas by modifying pre-existing wesnoth images
+    I release these under CC BY 4.0
+
+shadow-mage-halo
+    copied from the pre-existing wesnoth images in liberty
+
+
+//--------------------
+// ICONS
+//--------------------
+banana
+drowning
+    made by dalas
+    I release these under CC BY 4.0
+
+berserker
+brains
+cage
+coins
+escape-hatch
+ninja
+ogre
+sovereign
+    made by dalas by modifying pre-existing wesnoth images
+    I release these under CC BY 4.0
+
+clock (keith333) - CC BY 3.0
+    https://opengameart.org/content/clocks-watches-seamless-texture-with-normalmap-2
+    modified by dalas; I release my modified image under CC BY 4.0
+
+fiddle (IgnasD) - CC BY SA 3.0
+    https://opengameart.org/content/violin-0
+    modified by dalas; I release my modified image under CC BY 4.0 (or CC BY SA 3.0 if that's required by the original license)
```

Comment:
As I've already commented, someone needs to do a full audit of all of these images, but this comment stands out.

If you do not know whether CC BY SA 3.0 can be relicensed to [one of the CC licenses that's accepted in mainline](https://wiki.wesnoth.org/Wesnoth:Copyrights), then please ask first before putting it into a 700 file PR.

While writing this comment, the extra time spent reading it has made me realise that this is a BY SA to BY relicense, which is a separate issue to the question of whether CC BY SA 3.0 allows relicensing to CC BY SA 4.0

Reply:
Sorry about all these license issues, I should have checked more carefully.  Will it cover things if I say _"I give permission for the wesnoth team to license any of my EI contributions under whatever license they want"_?

Reply:
Oof, I see.  In this case, does CC BY-SA 3.0 allow the new contributions to be licensed under CC BY-SA 4.0?   Or is it stuck at 3.0 and I should create a replacement for the icon?

Reply:
3.0 and 4.0 are distinct licenses, so no.

Reply:
Ok.  I think this fiddle specifically actually isn't used anymore - Yumi didn't like the achievement it was originally for - but I can create from-scratch images to replace anything else that has licensing concerns.

(so this particular image can be deleted, and let me know which other ones I should replace)

Reply:
I mean, all of them that don't comply with the licensing Wesnoth requires. So everything that's not GPL v2+ or CC BY-SA 4.0 licensed. I'm not planning on listing all of them out though - if you're submitting images for inclusion then you should know whether they meet the license requirements or not.